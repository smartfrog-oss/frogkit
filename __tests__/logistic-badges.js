import { mount } from 'vue-test-utils'
import LogisticBadges from '@/components/logistic-badges'

describe('Logistic Badges component', () => {

  it('should render component and match snapshot', () => {
    const wrapper = mount(LogisticBadges)
    expect(wrapper.html()).toMatchSnapshot()
  })

})
