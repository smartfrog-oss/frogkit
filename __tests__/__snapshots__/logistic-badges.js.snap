// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logistic Badges component should render component and match snapshot 1`] = `<flex align="center" justify="space-around"><img width="142px" src="../assets/images/dhl.svg" alt="dhl"> <img width="142px" src="../assets/images/gogreen.svg" alt="gogreen"></flex>`;
